packages/nimble-components/src/anchor-tree-item/index.ts

```diff
@@ -0,0 +1,171 @@
+import { attr } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    AnchorOptions,
+    StartEnd,
+    applyMixins,
+    isTreeItemElement,
+    TreeItem as FoundationTreeItem
+} from '@microsoft/fast-foundation';
+import { keyArrowLeft, keyEnter } from '@microsoft/fast-web-utilities';
+import { AnchorBase } from '../anchor-base';
+import { styles } from './styles';
+import { template } from './template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-tree-item': AnchorTreeItem;
+    }
+}
+
+/**
+ * A nimble-styled anchor tree item
+ */
+export class AnchorTreeItem extends AnchorBase {
+    /**
+     * When true, the control will appear selected by user interaction.
+     * @public
+     * @remarks
+     * HTML Attribute: selected
+     */
+    @attr({ mode: 'boolean' })
+    public selected = false;
+
+    /**
+     * When true, the control will be immutable by user interaction. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled | disabled HTML attribute} for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: disabled
+     */
+    @attr({ mode: 'boolean' })
+    public disabled = false;
+
+    /**
+     * Whether the item is focusable
+     *
+     * @internal
+     */
+    public focusable?: boolean;
+
+    /**
+     * Indicates if the tree item is nested
+     *
+     * @internal
+     */
+    public nested?: boolean;
```

Comment:
Is this used? This seems redundant to `isNestedItem`.

Reply:
This is set by the parent FAST tree view. I'm not sure if the value is actually used, and I also don't see `isNestedItem` being used. Still, I feel more comfortable leaving them both in since that is what the FAST tree item has.

Reply:
After closer examination, this is only used by FAST for styling (fluent UI), but we don't need it. Removing.